dime-coding-standards/checklists/tex/hfcs_checklist.tex

```diff
@@ -0,0 +1,282 @@
+		
+		\section*{High-frequency checks}
+  This checklist lists important factors to consider while reviewing your code review partner's \textbf{high-frequency checks}. Please fill this checklist, and submit it as an attachment when you submit \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed form.}
+
+  \hfill \break 
+  \textbf{Note:} We are piloting this checklist as part of improving the code review process. The questions in this list are therefore \textbf{not required}, and we welcome your feedback on the points listed below.
+
+  \subsection*{General}
+		
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{The data shared does not contain any personally-identifying information (PII).}
+
+                \item[] \dimeCheckBox{The code creates a tracking sheet (or dashboard) to track progress of the survey.}
+        \end{itemize}
+
+ \hfill \break 
+The following sections cover various aspects of the \textbf{contents of the high-frequency checks} included in the package, but not all may apply to the high-frequency checks you are reviewing.
+
+\subsection*{Survey progress}
+Tracking \textbf{survey progress} involves checks for the following:
+            \begin{itemize}
+
+            \item[]\dimeCheckBox{\textbf{Response rates -}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[] \dimeCheckBox{Table on number/share of surveys attempted, completed, and declined.}
+                
+                \item[] \dimeCheckBox{Bar graphs for surveys attempted, completed, and declined - for each day}
+                
+                \item[] \dimeCheckBox{Graph for total surveys attempted, completed, and declined - on aggregate, and/or over last \textbf{X} days}
+                
+                \item[] \dimeCheckBox{Bar graph for reasons for declining (categorical)}
+                
+            \end{itemize}
+
+
+            \item[]\dimeCheckBox{\textbf{Duration -}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[] \dimeCheckBox{Bar graph of average duration for each day}
+                
+                \item[] \dimeCheckBox{Distribution of duration (either as k-desnity curves, or as box-plots by modules/sections}
+                
+            \end{itemize}
+
+            \end{itemize}
+            
+
+\subsection*{Survey quality}
+Tracking \textbf{survey quality} involves checks for the following:
+            \begin{itemize}
+            
+            \item[]\dimeCheckBox{\textbf{Duplicates -}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[] \dimeCheckBox{Spreadsheet listing duplicate entries in ID and non-ID variables to share with the field team for corrections}
```

Comment:
Does "non-ID variables to" refer to the list of non-ID variables for which two duplicates have different values? The thing `iecompdup` and `ieduplicates` do?

If it is, consider making this two distinct points. As listing duplicates in ID variables lists duplicates, and the other provide info on how to resolve them. They are very much related but you the first one can be done without the second, even though we recommend doing both. 
```suggestion
                \item[] \dimeCheckBox{Spreadsheet listing duplicate entries in ID}
                \item[] \dimeCheckBox{Spreadsheet listing some information to be used to resolve duplicates, such as listing non-ID variables for which duplicates has different values}
```

Reply:
agree. one point is whether they are checking for duplicates, the other is whether they are exporting duplicates so that they can be reviewed in the field.